resources/build/Makefile

```diff
@@ -0,0 +1,63 @@
+default: docker-image
```

Comment:
[fix] SPDX license identifier and copyright.

Reply:
c4c6231033860c9dd933a144acc33d7d8622a7f5